Yona/Yona/Settings/SettingsViewController.swift

```diff
@@ -110,7 +122,12 @@ extension SettingsViewController:UITableViewDelegate {
     }
     
     func tableView(tableView: UITableView, didSelectRowAtIndexPath indexPath: NSIndexPath) {
-        if indexPath.row == 2 {
+        if indexPath.row == 0 {
+            //change pin
+        } else if indexPath.row == 1 {
+            //privacy
+            performSegueWithIdentifier("privacyStatement", sender: self)
+        } else if indexPath.row == 2 {
             callAddDeviceMethod()
         }
         else if indexPath.row == 3 {
```

Comment:
I have commented on these line in PR #119 

What does the different indexPath.row numbers supposed to mean?
The indexes are meaningless in such situations, you can't guess the functionality by looking at the numbers. There are several ways to deal with such thing, but since it seems there are only 3 cases, we can handle them like that
`let setting = settingsArray[indexPath.row]

    if setting == NSLocalizedString("change-pin", comment: ""){

    }else if setting ==  NSLocalizedString("privacy", comment: ""){

    }else if setting == NSLocalizedString("delete-user", comment: ""){

    }`
At least it now contains words instead of numbers, which can help you:
1- Easily guess what this code supposed to do
2- It is not depending on the order any more.